BrowserView/lib/schema.js

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/**
+ *
+ * Interface to the JSON Schema processor package "ajv". It is used to check
+ * the nicknames' file.
+ *
+ * (This is a somewhat cut-back version of the general 'schemas.js' file that takes the schema from a local file and
+ * also checks the configuration file).
+ *
+ */
+
+// eslint-disable-next-line quotes
+const nicknames_schema = JSON.parse(`{
```

Comment:
Let's make this a `require()` so we can avoid managing the JSON in code (i.e. folks can still benefit from the standalone JSON version--which we can iterate separately as needed).

See https://stackoverflow.com/a/7165572/507169 for examples.

Reply:
I must admit I did not know about this trick. It is neat; I have replaced the current one with this. Thx!